include/odp/api/spec/ipsec.h

```diff
@@ -146,6 +146,13 @@ typedef struct odp_ipsec_inbound_config_t {
 	 */
 	odp_proto_chksums_t chksums;
 
+	/** Drop TFC dummy packets (NH=59) instead of reporting it via
+	 *  @tfc_dummy_packet error bit.
+	 *
+	 *  @see tfc_dummy_packets_drop
+	 */
+	odp_bool_t tfc_dummy_packets_drop;
```

Comment:
The documentation of odp_ipsec_in() says: "Outputted packets with error status have not been transformed but the original packet is returned."

Maybe it is too much promised for TFC packets (and the application should not need the original packet in that case anyway). It would anyway be good to clarify in the comments how exactly odp_ipsec_in() behaves with TFC packets.